src/App.tsx

```diff
@@ -61,13 +62,19 @@ export const App = (): React.ReactElement => {
                   </NavLink>
                 </Nav.Item>
                 <Nav.Item>
-                  <NavLink
-                    to="/paper"
-                    className="nav-link"
-                    activeClassName="active"
-                  >
-                    White Paper
-                  </NavLink>
+                  <NavDropdown title="White Paper" id="basic-nav-dropdown">
+                    <NavDropdown.Item href="/paper">
```

Comment:
I wonder if active item marking is as simple as removing the href here and adding a `<NavLink>` child? Not a blocker if it's not, but would be nice to have.

Reply:
I tried out that suggestion and I couldn't get it working. I am brand new to React, so that's that main issue. ;) Maybe someone else can take a stab at it?